### PR TITLE
Fix logout in AWS auth manager

### PR DESCRIPTION
**Pull request by vincbeck**
The logout link in Airflow is actually a form which then generate a POST request

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: tests/providers/amazon/aws/auth_manager/views/test_auth.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def aws_app():
         ) as mock_is_policy_store_schema_up_to_date:
             mock_is_policy_store_schema_up_to_date.return_value = True
             mock_parser.parse_remote.return_value = SAML_METADATA_PARSED
-            return application.create_app(testing=True)
+            return application.create_app(testing=True, config={"WTF_CSRF_ENABLED": False})
```
Comment (o-nikolas) [inline tests/providers/amazon/aws/auth_manager/views/test_auth.py:72]: Why was this change required here?

**Review by o-nikolas — APPROVED** [review]

File: tests/providers/amazon/aws/auth_manager/views/test_auth.py (hunk 1 above)
Comment (vincbeck) [reply tests/providers/amazon/aws/auth_manager/views/test_auth.py:72]: Flask require a csrf token when submitting a post request. Setting this constant to `False` disable this check. Which we want for tests :)
